react/Chip/styles.styl

```diff
@@ -1,16 +1,21 @@
 @require '../../stylus/components/chip'
 
 .Chip
+.c-chip
     @extend $chip
 
 .RoundChip
+.c-chip--round
     @extend $round-chip
 
 .ChipSeparator
+.c-chip-separator
     @extend $chip-separator
 
 .ChipButton
+.c-chip-button
     @extend $chip-button
 
 .ChipButton--disabled
+.c-chip-button--disabled
```

Comment:
We should remove the `ChipButton` classes (and thus modify the ChipButton jsx), this way there is no duplication :)

Reply:
Yes, It's just that this component is done differently and I'm not sure I understand it all. I tried something and failed. So I figured it would have to be done later. But maybe I can give it another try.

Reply:
Nevermind… I found why I could do it. It's done :)